### ISSUE TITLE
[Android] Refactoring Mistake in ReactNativePermissionsModule

### ISSUE THREAD

**Issue by norman-kapschefsky** [zoontek/react-native-permissions#45]
Hi,
within [ReactNativePermissionsModule](https://github.com/yonahforst/react-native-permissions/tree/master/android/src/main/java/com/joshblour/reactnativepermissions/ReactNativePermissionsModule.java) in Line 96 

```
mPermissionsModule.requestPermission(permission, new PromiseImpl(resolve, reject));
```

you missed to replace `PromiseImpl` parameter with single`resolve` and `reject` callback. It wont build on Android in this way right now. Cheers.

**Comment by yonahforst** [comment]
@norman-kapschefsky thanks for catching this. I'm busy on another project, and probably won't be able to address this until January '17. I'm happy to merge a PR, if anyone has the time to submit one!

**Comment by yonahforst** [comment]
fixed in #48
